### ISSUE TITLE
ERROR in Metadata version mismatch for module /home/abhishek/cms/web/node_modules/@swimlane/ngx-dnd/release/index.d.ts, found version 4, expected 3,

### ISSUE THREAD

**Issue by abhishekpandeykr** [swimlane/ngx-dnd#71]
I am getting this issue while running ng serve, my angular version is 4.3.5
my package.json file is:
```
{
  "name": "cms",
  "version": "0.0.0",
  "license": "MIT",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@agm/core": "1.0.0-beta.1",
    "@angular/animations": "4.3.5",
    "@angular/common": "4.3.5",
    "@angular/compiler": "4.3.5",
    "@angular/core": "4.3.5",
    "@angular/forms": "4.3.5",
    "@angular/http": "4.3.5",
    "@angular/material": "2.0.0-beta.4",
    "@angular/platform-browser": "4.3.5",
    "@angular/platform-browser-dynamic": "4.3.5",
    "@angular/router": "4.3.5",
    "@swimlane/ngx-charts": "6.1.0",
    "@swimlane/ngx-datatable": "9.3.1",
    "@swimlane/ngx-dnd": "^3.2.0",
    "@types/jquery": "2.0.48",
    "angular-font-awesome": "2.0.3",
    "angular2-websocket": "0.9.5",
    "core-js": "2.5.0",
    "font-awesome": "4.7.0",
    "graceful-fs": "4.1.11",
    "hammerjs": "2.0.8",
    "jquery": "2.2.4",
    "medium-editor": "5.23.2",
    "moment": "2.18.1",
    "ng2-file-upload": "1.2.1",
    "ng2-translate": "5.0.0",
    "ngx-bootstrap": "1.9.1",
    "rxjs": "5.4.3",
    "underscore": "1.8.3",
    "zone.js": "0.8.16"
  },
  "devDependencies": {
    "@angular/cli": "1.3.1",
    "@angular/compiler-cli": "4.3.5",
    "@types/googlemaps": "^3.29.1",
    "@types/jasmine": "2.5.38",
    "@types/node": "6.0.88",
    "codelyzer": "2.0.1",
    "jasmine-core": "2.5.2",
    "jasmine-spec-reporter": "3.2.0",
    "karma": "1.4.1",
    "karma-chrome-launcher": "2.0.0",
    "karma-cli": "1.0.1",
    "karma-coverage-istanbul-reporter": "0.2.3",
    "karma-jasmine": "1.1.0",
    "karma-jasmine-html-reporter": "0.2.2",
    "protractor": "5.1.2",
    "ts-node": "2.0.0",
    "tslint": "4.5.1",
    "typescript": "2.2.2"
  }
}
```

**Comment by adriangube** [comment]
Downgrade and fix version to 3.1.0 has worked for me.

**Comment by Hypercubed** [comment]
@abhishekpandeykr @adriangube Can you please give the next version a try: `npm install @swimlane/dragula @swimlane/ngx-dnd@next`

**Comment by Hypercubed** [comment]
Just published 5.0.2, if you still see this issue please reopen.

**Comment by alesce** [comment]
Angular 4.4.6

Same problem with ngx-dnd version 4.0.1, 4.0.2, 5.0.1, 5.0.2 and the latest 5.1.0

Any ideas?
